File: content/search/source/includes/index/field-types/number/code-snippets/csharp/CreateIndexExampleIntegers.cs
using MongoDB.Bson;
using MongoDB.Driver;

// connect to your Atlas deployment
var uri = "<connection-string>";

var client = new MongoClient(uri);

var db = client.GetDatabase("sample_analytics");
var collection = db.GetCollection<BsonDocument>("accounts");

// define your MongoDB Search index
var index =  new CreateSearchIndexModel(
  "default", new BsonDocument
  {
    { "mappings", new BsonDocument
      {
        { "dynamic", false },
        { "fields", new BsonDocument
          {
            { "account_id", new BsonDocument
              {
                { "type", "number" },
                { "representation", "int64" },
                { "indexDoubles", false }
              }
            }
          }
        }
      }
    }
  });

var result = collection.SearchIndexes.CreateOne(index);
Console.WriteLine($"New index name: {result}");

File: content/search/source/includes/index/field-types/number/code-snippets/nodejs/create-index-example-integers.js
const { MongoClient } = require("mongodb");

// connect to your Atlas deployment
const uri =  "<connection-string>";

const client = new MongoClient(uri);

async function run() {
  try {

    // set namespace
    const database = client.db("sample_analytics");
    const collection = database.collection("accounts");

    // define your MongoDB Search index
    const index = {
        name: "default",
        definition: {
            /* search index definition fields */
            "mappings": {
                "fields": {
                    "account_id": {
                        "type": "number",
                        "representation": "int64",
                        "indexDoubles": false
                    }
                }
            }
        }
    }

    // run the helper method
    const result = await collection.createSearchIndex(index);
    console.log("New index name: " + result);
  } finally {
    await client.close();
  }
}

run().catch(console.dir);